### ISSUE TITLE
Outdated Github page on swift-docc

### ISSUE THREAD

**Issue by Kyle-Ye** [apple/swift-docc#392]
### Description

Outdated Github Page doc on swift-docc, swift-markdown and other swift-docc projects repo.

https://apple.github.io/swift-docc/

https://apple.github.io/swift-markdown/

Although we add /bin/update-gh-pages-documentation-site script and the Github Deployment support. But it was triggered/deployed only once. While it should be trigered everytime a new PR is merged into main IMO.

<img width="1077" alt="image" src="https://user-images.githubusercontent.com/43724855/192334700-c751ecfb-f09b-4b6f-b813-ce7e0221b502.png">

### Checklist

- [X] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/apple/swift-docc/issues).

### Expected Behavior

Github deployment should be trigered everytime a new PR is merged into main.

### Actual behavior

It was only trigger once manully.

**Comment by Kyle-Ye** [comment]
cc @ethan-kusters @franklinsch

**Comment by ethan-kusters** [comment]
@shahmishal is this something we could do in CI potentially?

**Comment by Kyle-Ye** [comment]
> @shahmishal is this something we could do in CI potentially?

+1 for this. Maybe we can do it in Swift CI or via Github action

**Comment by Kyle-Ye** [comment]
> @shahmishal is this something we could do in CI potentially?

Ping @shahmishal
